src/common/resources/exportConfigs.js

```diff
@@ -0,0 +1,20 @@
+import { baseManifest } from '@folio/stripes-acq-components';
+
+import {
+  EXPORT_CONFIGURATIONS_API,
+} from '../constants';
+
+export const exportConfigsResource = {
+  ...baseManifest,
+  path: EXPORT_CONFIGURATIONS_API,
+  records: 'exportConfigs',
+  accumulate: true,
+  fetch: false,
+};
+
+export const credentialsResource = {
+  ...baseManifest,
+  accumulate: true,
+  fetch: false,
+  path: `${EXPORT_CONFIGURATIONS_API}/%{exportConfigId.id}/credentials`,
```

Comment:
someone will include this resource in another component and it might take a long time to understand, why it doesn't work. That's why I've put my concern to move `path` with relation to some local resource name out of common code

Reply:
Moved `path` to local resource